api/data_explorer/controllers/export_url_controller.py

```diff
@@ -116,10 +116,6 @@ def _get_table_and_clause(es_field_name, field_type, value, bucket_interval,
     eg "((age76 >= 20 AND age76 < 30) OR (age76 >= 30 AND age76 < 40))".
     """
     sample_file_type_field = False
-    if field_type == 'samples_overview':
-        es_field_name = current_app.config['FACET_INFO']['Samples Overview'][
-            'elasticsearch_field_names'][value]
-        value = True
```

Comment:
The `_has_` fields on line 123 is only used for Samples Overview right?

If so, please delete logic around `_has_`. (These fields should be deleted in indexer repo as well.)

Reply:
We're using the _has fields for AMP PD in their own facets, so I'd like to keep this here and in the indexer.